frontend/src/metabase-lib/lib/Dimension.js

```diff
@@ -841,8 +843,10 @@ export class AggregationDimension extends Dimension {
 
   column(extra = {}) {
     const aggregation = this.aggregation();
+    const { special_type, ...column } = super.column();
     return {
-      ...super.column(),
+      ...column,
+      ...(!UNAGGREGATED_SPECIAL_TYPES.has(special_type) && { special_type }),
```

Comment:
A similar quick comment on why we look for the `UNAGGREGATED_SPECIAL_TYPES`  here would be helpful.